api/v1/views/index.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python3
+"""import app_views from api.v1.views
+create a route /status on the object app_views that returns a JSON: status:
+    OK (see example)"""
+from flask import Flask, Blueprint, jsonify
+from api.v1.views import app_views
+from models import storage
+from models.amenity import Amenity
+from models.city import City
+from models.place import Place
+from models.review import Review
+from models.state import State
+from models.user import User
```

Comment:
These are unnecessary because when you call storage, the storage file has all of them already